resources/views/people/edit.blade.php

```diff
@@ -98,7 +98,7 @@
             <form-select
               :options="{{ $genders }}"
               value="{{ $contact->gender_id }}"
-              :required="true"
+              :required="false"
```

Comment:
Editing a gender and setting a "blank" gender now does not work. It triggers an error saying the field is mandatory.

![image](https://user-images.githubusercontent.com/61099/54732091-3e92c980-4b68-11e9-9f1f-bad225e93911.png)

Reply:
Fixed